packages/wallet/src/data/services/market-proxy/index.ts

```diff
@@ -0,0 +1,911 @@
+/**
+ * @see https://docs.coingecko.com/ for CoinGecko API documentation
+ * @see https://test.market.status.im for proxy endpoint
+ *
+ * This module uses CoinGecko API through a proxy at https://test.market.status.im
+ * Example: https://pro-api.coingecko.com/api/v3/coins/list -> https://test.market.status.im/v1/coins/list
+ */
```

Comment:
non-blocking: would call it or move it to `coingecko/` as well then, like `alchemy/` stayed "alchemy" even though proxied

Reply:
merged the market-proxy with the coingecko directory